tests/cql/CqlIntervalOperatorsTest.xml

```diff
@@ -251,14 +251,16 @@
 		<test name="ExpandPer1">
 			<expression>expand { Interval[10.0, 12.5] } per 1</expression>
 			<output>{ Interval[10, 10], Interval[11, 11], Interval[12, 12] }</output>
+			<!-- Translation Error: Could not resolve call to operator Expand with signature (list<interval<System.Decimal>>,System.Integer). -->
 		</test>
 		<test name="ExpandPerMinute">
 			<expression>expand { Interval[@T10, @T10] } per minute</expression>
 			<output>{ }</output>
 		</test>
 		<test name="ExpandPer0D1">
 			<expression>expand { Interval[10, 10] } per 0.1</expression>
-			<output>{ Interval[10.0, 10.0], Interval[10.1, 10.1], ..., Interval[10.9, 10.9] }</output>
+			<output>{ Interval[10.0, 10.0], Interval[10.1, 10.1], Interval[10.2, 10.2], Interval[10.3, 10.3], Interval[10.4, 10.4], Interval[10.5, 10.5], Interval[10.6, 10.6], Interval[10.7, 10.7], Interval[10.8, 10.8], Interval[10.9, 10.9] }</output>
```

Comment:
The `...` doesn't translate at all (for obvious reasons).  To support automated approaches to ingesting these test files, we need to spell the whole thing out.